README.adoc

```diff
@@ -34,6 +34,7 @@ buildPlugin(
 * `repo` (default: `null`  inherit from Multibranch) - custom Git repository to check out
 * `useContainerAgent` (default: `false`) - uses a link:https://github.com/jenkins-infra/documentation/blob/main/ci.adoc#container-agents[Container agent] instead of a Virtual Machine: usually faster to start and generates less costs for the project
 ** Please note that the implementation of "containers" can be changed over time
+* `forkCount` (default: null) - Execute tests with `forkCount` Java virtual machines. Common values include `0.45C`, `1`, `2`, and `1C` (if you terminate the value with a 'C', that value will be multiplied by the number of available CPU cores).
```

Comment:
May be helpful to note that the default of `null` means to use Surefire’s default, which would normally be the same as `1`: one test at a time, run serially.

Reply:
Very good point.

Reply:
done in 942490c939a853e94f38f43fcfe9df1776dfafc3